index.js

```diff
@@ -1 +1,24 @@
-// Do work!
\ No newline at end of file
+const { expect } = require('chai')
```

Comment:
`expect` is not used. We don't need it to be declared here.  This might be included in a test file normally.